src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -4198,8 +4198,9 @@ int SketchObject::split(int GeoId, const Base::Vector3d& point)
 
                 // TODO: Do we apply constraints on center etc of the conics?

```

Comment:
Does this change answer this TODO - so remove it?

Reply:
There's also a question of whether there should be a constraint between the two new pieces.

Reply:
As Ajinkya said, this is another question.

Reply:
Ah, my error. NVM